index.html

```diff
@@ -1 +1,192 @@
+          </div>
+          <div id="share-buttom" class="video-buttoms">
+              <form class="" action="index.html" method="post">
+                  <button type="submit">
+                      <img src="https://cdn.shopify.com/s/files/1/0099/9562/products/274_Hero_Social_Sharing_Button_2048x2048.jpg?v=1475281658" alt="" id="share-buttom-img">
+                  </button>
+              </form>
+          </div>
+          </div>
+          <div class="user-panel-x">
+              <div id="user-icon">
+                <img src="https://blog.optimizely.com/wp-content/uploads/2014/10/codecademy-logo-e1413249893959.png" alt="">
+                <div id="Username"><p>LearnCode.academy<br> <ins class="date">Posted on Jan 16 2014.</ins></p></div>
+                <div id="description"><p>Github Tutorial For Beginners - learn Github for Mac or Github for windows
+                    If you've been wanting to learn Github, now's the perfect time!  Github is seen as a big requirement
+                    by most employers these days and is very critical to business workflow.
+                    This Github tutorial will cover the basics of how to use Github and the command line.</p></div>
+
+              </div>
+              <div id="comments">
+                  <p id="words">Add a comment:</p>
+                  <textarea></textarea>
+                  <div class="comment">
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+
+                  </div><br>
+                  <div class="comment">
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+                          <p>Comment</p><br>
+
+                  </div><br>
+                  <div class="comment">
```

Comment:
## Found a :beetle: Here!

This requires your attention, all of these  `"comment"` class `divs` need to have a little adjustment, with the current way they are styled we get a vertical scrolling var in every container, now this might be because of certain constraints set by CSS, like `height` property, please kindly go over your `CSS` code and your `HTML`, and find what's causing that distress.

Reply:
I wrote multiple line texts into each div just to be sure if they can be automatically scroll-able.Each div has one comment text which are few line texts so if i remove the text in those divs , they turn into normal divs without scrolls. Am i wrong ?

Reply:
Also we got confused about workflow and made things wrong at the first time. Merged the additional branch which we worked on, into master branch. That is why we opened up a new repo and upload our work into a new branch and NOT merge it with master branch ^^ So only my commits are seem but actually we were pretty good about workflow except merging ^^